server/v2/tests/mockData/entryData.js

```diff
@@ -0,0 +1,31 @@
+import Auth from '../../helpers/authenticate';
+import authData from './authData';
+
+
+const validTokens = {
+    savedUser: Auth.generateToken(authData[0].email, 1, false),
+    unsavedUser: Auth.generateToken('gakwererepacis@gmail.com', 1200, false),
+    noEntryUser: 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJlbWFpbCI6InJvYmVydHJ0QGdtYWlsLmNvbSIsImlkIjoyLCJpYXQiOjE1NzI3NzIwOTB9.oZhkuh1fjUM-pEAKzxGskzbRPWLDES4LVOtXMLh9moI'
+};
```

Comment:
@Nigorjeanluc, Imagine a case where you are not using numerical IDs (let's say you are using UUIDs) how will you be sure that this hardcoded id is the same as the one stored in the database?  To create a toke that will be used in your tests you need to use the tests hooks like ```before or beforeEach``` and hit the login route or the signup route. you just hit whatever returns a token. from there you can have access to that token.  feel free to reach out to me in case you need help.